tests/backup/backup_helper.go

```diff
@@ -190,6 +198,62 @@ func CreateBackup(backupName string, clusterName string, bLocation string, bLoca
 	log.Infof("Backup [%s] created successfully", backupName)
 	return nil
 }
+
+// CreateBackupAndGetBackupObjects returns a BackupContext
+func CreateBackupAndGetBackupCtx(backupName string, clusterName string, bLocation string, bLocationUID string,
```

Comment:
There is no need for this function. You can use the usual `CreateBackup` function which will return once the backup is successfully done. Then you can call your `ValidateBackup` function which will only accept the `backupName`. The computation of getting the backup UID and getting the inspect response can be done inside `ValidateBackup` itself

Reply:
As part of an upcoming PR on deeper backup validation in `CreateBackup` this code will be used. Basically, this function will replace the current `CreateBackup`. So, this extra function is just temporary as I wanna use it _right now_